modules/nginx/manifests/config/vhost/proxy.pp

```diff
@@ -149,14 +149,15 @@
   statsd::counter { "${counter_basename}.http_500": }
 
   @@icinga::check::graphite { "check_nginx_5xx_${title}_on_${::hostname}":
-    ensure    => $ensure,
-    target    => "transformNull(stats.${counter_basename}.http_5xx,0)",
-    warning   => $alert_5xx_warning_rate,
-    critical  => $alert_5xx_critical_rate,
-    from      => '3minutes',
-    desc      => "${title} nginx 5xx rate too high",
-    host_name => $::fqdn,
-    notes_url => monitoring_docs_url(nginx-5xx-rate-too-high-for-many-apps-boxes),
+    ensure                   => $ensure,
+    target                   => "transformNull(stats.${counter_basename}.http_5xx,0)",
+    warning                  => $alert_5xx_warning_rate,
+    critical                 => $alert_5xx_critical_rate,
+    from                     => '30minutes',
+    desc                     => "${title} nginx 5xx rate too high",
```

Comment:
Minor: Can we make the `desc` consistent between these two files? (Especially if they're pointing to the same monitoring docs URL).

Reply:
Sure, I've gone with the licensify one, and updated the docs to match: https://github.com/alphagov/govuk-developer-docs/pull/2681